api4/system.go

```diff
@@ -384,3 +385,33 @@ func getAnalytics(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	w.Write([]byte(rows.ToJson()))
 }
+
+func testS3(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Oh actually, can we get a unit test for this too?

Reply:
Sorry, should have pointed that out in my first review

Reply:
sure thing I forgot to add, my fault

Reply:
@jwilander added